displays/cavity_display/backend/fault.py

```diff
@@ -137,7 +138,7 @@ def get_fault_count_over_time_range(
         for archiver_value in data_handler.value_list:
             try:
                 if self.is_faulted(archiver_value):
-                    counter.fault_count += 1
+                    counter.alarm_count += 1
```

Comment:
Do we need to account for warning_count some where here? I see a `counter.alarm_count`, `counter.ok_count`, and `counter.invalid_count`. Or is the entirety of warning counts being handled in `get_fault_counts` in backend_cavity.py?

Reply:
Oh yeah this function just isn't being used at all right now lol